modules/gitops-configuring-high-availability-for-argo-rollouts.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enabling-ha-support-for-argo-rollouts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-high-availability-for-argo-rollouts_{context}"]
+= Configuring high availability for Argo Rollouts
+
+To enable high availability, configure the `ha` specificaiton in the `RolloutManager` CR by completing the following steps:
+
+.Procedure
+
+.  Log in to the {OCP} web console as a cluster administrator. 
+
+. In the *Administrator* perspective, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to create and configure a `RolloutManager` custom resource (CR) from the *Project* drop-down menu.
+
+. Select *{gitops-title}* from the installed operators.
+
+. In the *Details* tab, under the *Provided APIs* section, click *Create instance* in the *RolloutManager* pane.
+
+. On the *Create RolloutManager* page, select the *YAML view* and edit the YAML.
++
+.Example for enabling the `ha` field in the `RolloutManager` CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: argo-rollouts
+  namespace: openshift-gitops
+spec:
+  ha:
+    enabled: true #<1>
+----
+<1> Specifies whether high availability is enabled or not. If the value is set to `true`, high availability is enabled.
+
+. Click *Create*.
+
+. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the RolloutManager instance shows as *Phase: Available*.
+
+. Verify the status of the Rollouts deployment by completing the following steps:
+.. In the *Administrator* perspective, click *Workloads* -> *Deployments*.
+.. Click the *argo-rollouts* deployment.
++
+As a result, the number of replicas in the Rollouts Deployment is set to 2 in the *Details* tab. Also, the following configuration is displayed in the *YAML* tab.
++
+.Example for the Argo Rollouts deployment for enabling high availability in the Deployments page
```

Comment:
I think this title is a bit too long. I'd suggest something simpler, like:

```suggestion
.Example Argo Rollouts deployment configuration file
```

Reply:
Fixed.